### PR TITLE
Fix golint errors for pkg/registry/apps/statefulset/storage

### PR DESCRIPTION
**Pull request by RaunakShah**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fix golint errors for pkg/registry/apps/statefulset/storage

**Which issue(s) this PR fixes**:
Part of https://github.com/kubernetes/kubernetes/issues/68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/registry/apps/statefulset/storage/storage.go [hunk 1]
```diff
@@ -46,6 +46,7 @@ type StatefulSetStorage struct {
 	Scale       *ScaleREST
 }
 
+// NewStorage returns a new StatefulSetStorage object
```
Comment (yuxiaobo96) [inline pkg/registry/apps/statefulset/storage/storage.go]:
```suggestion
// NewStorage returns a new StatefulSetStorage object.
```

File: pkg/registry/apps/statefulset/storage/storage.go [hunk 2]
```diff
@@ -59,7 +60,7 @@ func NewStorage(optsGetter generic.RESTOptionsGetter) (StatefulSetStorage, error
 	}, nil
 }
 
-// rest implements a RESTStorage for statefulsets against etcd
+// REST implements a RESTStorage for statefulsets against etcd
```
Comment (yuxiaobo96) [inline pkg/registry/apps/statefulset/storage/storage.go]:
```suggestion
// REST implements a RESTStorage for statefulsets against etcd.
```

File: pkg/registry/apps/statefulset/storage/storage.go [hunk 3]
```diff
@@ -100,6 +101,7 @@ type StatusREST struct {
 	store *genericregistry.Store
 }
 
+// New returns a new statefulset object
```
Comment (yuxiaobo96) [inline pkg/registry/apps/statefulset/storage/storage.go]: ditto

File: pkg/registry/apps/statefulset/storage/storage.go [hunk 4]
```diff
@@ -124,6 +126,7 @@ func (r *REST) ShortNames() []string {
 	return []string{"sts"}
 }
 
+// ScaleREST implements the REST endpoint for scaling a statefulSet
```
Comment (yuxiaobo96) [inline pkg/registry/apps/statefulset/storage/storage.go]: ditto

File: pkg/registry/apps/statefulset/storage/storage.go [hunk 5]
```diff
@@ -132,6 +135,7 @@ type ScaleREST struct {
 var _ = rest.Patcher(&ScaleREST{})
 var _ = rest.GroupVersionKindProvider(&ScaleREST{})
 
+// GroupVersionKind creates the "scale" kind for the input group version
```
Comment (yuxiaobo96) [inline pkg/registry/apps/statefulset/storage/storage.go]: ditto

**Review by yuxiaobo96 — COMMENTED** [review]
A little suggestion

**Comment by RaunakShah** [comment]
@yuxiaobo96 done, thanks for the suggestion!

**Pull request closed without merging**